osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacyKiaiGlow.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Skinning.Legacy
+{
+    internal partial class LegacyKiaiGlow : Container
+    {
+        public LegacyKiaiGlow()
+        {
+            AlwaysPresent = true;
+            Alpha = 0;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(ISkinSource skin)
+        {
+            Child = new Sprite
+            {
+                Texture = skin.GetTexture("taiko-glow"),
+                Origin = Anchor.Centre,
+                Anchor = Anchor.Centre,
+                Scale = new Vector2(0.75f),
```

Comment:
I'm not sure if it matches 1:1, but stable uses `0.7f` scale. It also applies a colour tint of `new Color(255, 228, 0, 0)`.

Reply:
Thanks. Regarding color, it seems like `Colour4` subtracts color from the original texture to match the requested values instead of "overlaying" the color on top. This makes the element appear less bright than it should be. Is it okay if I compensate for this in the fade in?

``` csharp
private const float colour_compensation = 1.58f;                // (255 * 3) / (255 + 228 + 0) = 1.58

this.FadeTo(colour_compensation, 180);
```

Reply:
You can't fade above `1.0f`, that makes no sense. A visual comparison would be a good starting point because I don't quite understand what you're trying to say.

Reply:
I can't reproduce the behavior I mentioned above anymore... Maybe restarting Visual Studio helped (hot reload shenanigans?). Either way it's working as expected now without fading above 1.